app/routes/user/email_notifications.server.ts

```diff
@@ -252,6 +252,6 @@ export class EmailNotificationVendingMachine {
       },
     })
 
-    await client.send(command)
+    return await client.send(command)
```

Comment:
I think that this will just raise an exception if it fails, right? No need to check the status code.